src/types.d.ts

```diff
@@ -0,0 +1,18 @@
+type ResponseDataType = {
+    page: number
+    perPage: number
+    total: number
+    totalPages: number
+    data: Array<UserType>
```

Comment:
It seems like this hardcoded type here makes it harder to maintain our code in the future. What if we want to have another `ResponseData`, but for `UserPets` instead of `Users`? We can use a generic type here to make it possible:

```ts
type ResponseData<T> = {
    page: number
    perPage: number
    total: number
    totalPages: number
    data: T
    support: {
        url: string
        text: string
    }
}

type UserResponseData = ResponseData<UserType[]>
```